src/scripts/PlayerBody.cs

```diff
@@ -343,7 +352,79 @@ public virtual void Update()
                         handTransforms[i].rotation = handsToFollow[i].rotation;
                     }
                 }
+
+                // Hide self when using quickbelt //
+                // TODO: better way to determine if this is the local PlayerBody?
+                if (headDisplayMode != HeadDisplayMode.Physical && optionAutoHideSelf)
+                {
+                    var usingQB = IsUsingQuickbelt();
+                    var pitch = headToFollow.rotation.eulerAngles.x;
+                    if (!selfIsHidden)
+                    {
+                        if (usingQB)
+                        {
+                            SetPOVBodyVisible(false);
+                            SetPOVHandsVisible(false);
+                            selfIsHidden = true;
+                        }
+                    }
+                    else if (!selfIsUnhiding)
+                    {
+                        if (pitch < SELF_UNHIDE_PITCH || (180 < pitch && pitch < 360))
+                        {
+                            selfIsUnhiding = true;
+                            StartCoroutine(nameof(SelfUnhideCoroutine));
```

Comment:
I'd prefer using AnvilManager.Run like vanilla, and to keep it consistent with how I've been running coroutines in the rest of the code